docs-utils/update-images.js

```diff
@@ -2,8 +2,9 @@ const fs = require('fs')
 const path = require('path')
 const frontmatter = require('front-matter')
 const imageDownloader = require('image-downloader')
+const { log } = require('console')
```

Comment:
This import is not used. It can be removed.

Reply:
Ok, removed.